Source/SA/RFmx LTE/RFmxLTE.cs

```diff
@@ -159,7 +159,7 @@ public static void ConfigureCommon(RFmxLteMX lte, CommonConfiguration commonConf
         #endregion
 
         #region Measurement Configuration
-        public static void ConfigureStandard(RFmxLteMX lte, SignalConfiguration signalConfig, string selectorString = "")
+        public static void ConfigureStandard(RFmxLteMX lte, StandardConfiguration signalConfig, string selectorString = "")

```

Comment:
```suggestion
        public static void ConfigureStandard(RFmxLteMX lte, StandardConfiguration standardConfig, string selectorString = "") 
```

probably need to do this everywhere the name changed, would you agree?

Reply:
Nice catch... my bad.